Adds dana's change and removes community (#4)
Pioneer-Valley-Housing-Coop/docs: one-pager.md

# Pioneer Valley Housing Co-op Seeks Property 

Are you interested in housing that fosters community, equity, sustainability, and stewardship for generations to come? What about a local alternative to renting and traditional homeownership? We are forming a group equity housing co-op to collectively invest in ourselves, our community, and the environment.

>  We aim to support community and resident wellbeing through being democratic and affordable through our structure as a group equity housing co-op. 

Author: Need some kinda transition right?

 
**Community** 
* Multiple homes clustered around common spaces and shared resources  
* Room for creative projects and enterprises
* Public access spaces and events

> 

Author: Love this as first thing! Very clear and practical

**Resident Wellbeing**
* Flexible commitment of renting
* Security and control of traditional homeownership

> * Community of cohousing or ecovillage without financial inaccessibility or cultural conformity/ too intentional? 

Author: Nice :100:

**Democratic** 
* Housing co-op will own assets as a group, with residents as equal voting members
* Collective decision making and conflict mediation

**Affordable** 
* [Group equity](https://www.nasco.coop/development/handbook/equity) model will ensure low move in share, to be returned when they move out
* Initial funds will be raised from grants and community investors, not residents

> * Residents will pay monthly costs to repay community investors

Author: I think this one could even start with a dumber down line, like... "Residents pay months costs to live there with low upfront costs, making it accessible to people who cannot afford a traditional home"?

We aim to live sustainably through community stewardship for a [just transition](https://movementgeneration.org/wp-content/uploads/2016/11/JT_booklet_English_SPREADs_web.pdf 

**Just**
* Housing and land should be decommodified and stewarded by and for the community, not speculative profits.
* Closed-loop, regenerative systems to support abundance and resilience away from an extractive economy.

**Sustainability**:
* Net-zero buildings using passive solar 
* Member of renewable energy co-op
* Food sovreignty through regenerative agriculture 

**Stewardship**: 

> * A community land trust will own the land permanently, and have a say in resale

Author: ```suggestion
* The community with have a permenant say in the land, through a community land trust owning it
```

Something like that? Or at least start with what the end result is, then get into particulars?
